### DISCUSSION TITLE
mdns?

### DISCUSSION THREAD

**Discussion by thesebastianf** [Blackymas/NSPanel_HA_Blueprint#1160] [General]
Hey,
mdns is for whatever reason not really working in my network.
so to see the status and flash wirelessly I need to set esphome to static ip.
can I just add it in the substition like this: ?

wifi:
  networks:
    - id: wifi_default
      ssid: ${wifi_ssid}
      password: ${wifi_password}
  power_save_mode: none
  ap:
    ssid: "${device_name}"
    password: ${wifi_password}
  manual_ip:
    static_ip: 192.168.1.12
    gateway: 192.168.1.195
    subnet: 255.255.252.0
    dns1: 192.168.1.195
    dns2: 192.168.1.195  

thanks!

**Comment by edwardtfn** [comment]
Please take a look at this: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/en/customization.md#manual-ip
